docs/updating_schemas.md

```diff
@@ -102,7 +117,7 @@ Refer to [object-lifecycle.md](https://github.com/IGVF-DACC/igvfd/blob/dev/docs/
                 schema = load_schema('encoded:schemas/treatment.json')
 
 
-5. Within in a class add in  *embedding*, *reverse links*, and *calculated properties* as necessary.
+6. Within in a class add in  *embedding*, *reverse links*, and *calculated properties* as necessary.
 
     * *Embedding* - specifying the properties embeded in the object when specifying ```frame=object```, for construct we have:
```

Comment:
This doesn't sound right. `frame=object` doesn't resolve links to embedded objects, `frame=embedded` does.

Reply:
Good catch.  I wonder if we should do a more fine toothed review of the whole doc.  I was focused on adding the new steps.